src/complex/Common/Constants.hpp

```diff
@@ -0,0 +1,190 @@
+
+inline constexpr float k_PiOver180F = k_PiOver180<float>;
+inline constexpr double k_PiOver180D = k_PiOver180<double>;
+
+inline constexpr float k_360OverPiF = k_360OverPi<float>;
+inline constexpr double k_360OverPiD = k_360OverPi<double>;
+
+inline constexpr float k_180OverPiF = k_180OverPi<float>;
+inline constexpr double k_180OverPiD = k_180OverPi<double>;
+
+inline constexpr float k_PiOver2F = k_PiOver2<float>;
+inline constexpr double k_PiOver2D = k_PiOver2<double>;
+
+inline constexpr float k_PiOver3F = k_PiOver3<float>;
+inline constexpr double k_PiOver3D = k_PiOver3<double>;
+
+inline constexpr float k_PiOver4F = k_PiOver4<float>;
+inline constexpr double k_PiOver4D = k_PiOver4<double>;
+
+inline constexpr float k_PiOver8F = k_PiOver8<float>;
+inline constexpr double k_PiOver8D = k_PiOver8<double>;
+
+inline constexpr float k_PiOver12F = k_PiOver12<float>;
+inline constexpr double k_PiOver12D = k_PiOver12<double>;
+
+inline constexpr float k_Sqrt2F = k_Sqrt2<float>;
+inline constexpr double k_Sqrt2D = k_Sqrt2<double>;
+
+inline constexpr float k_Sqrt3F = k_Sqrt3<float>;
+inline constexpr double k_Sqrt3D = k_Sqrt3<double>;
+
+inline constexpr float k_HalfSqrt2F = k_HalfSqrt2<float>;
+inline constexpr double k_HalfSqrt2D = k_HalfSqrt2<double>;
+
+inline constexpr float k_1OverRoot2F = k_1OverRoot2<float>;
+inline constexpr double k_1OverRoot2D = k_1OverRoot2<double>;
+
+inline constexpr float k_1OverRoot3F = k_1OverRoot3<float>;
+inline constexpr double k_1OverRoot3D = k_1OverRoot3<double>;
+
+inline constexpr float k_Root3Over2F = k_Root3Over2<float>;
+inline constexpr double k_Root3Over2D = k_Root3Over2<double>;
+
+inline constexpr float k_DegToRadF = k_DegToRad<float>;
+inline constexpr double k_DegToRadD = k_DegToRad<double>;
+
+inline constexpr float k_RadToDegF = k_RadToDeg<float>;
+inline constexpr double k_RadToDegD = k_RadToDeg<double>;
+
+inline constexpr float k_1Point3F = k_1Point3<float>;
+inline constexpr double k_1Point3D = k_1Point3<double>;
+
+inline constexpr float k_1Over3F = k_1Over3<float>;
+inline constexpr double k_1Over3D = k_1Over3<double>;
+
+inline constexpr float k_ACosNeg1F = k_ACosNeg1<float>;
+inline constexpr double k_ACosNeg1D = k_ACosNeg1<double>;
+
+inline constexpr float k_ACos1F = k_ACos1<float>;
+inline constexpr double k_ACos1D = k_ACos1<double>;
+
+inline constexpr float k_Tan_OneEighthPiF = k_Tan_OneEighthPi<float>;
+inline constexpr double k_Tan_OneEighthPiD = k_Tan_OneEighthPi<double>;
+
+inline constexpr float k_Cos_OneEighthPiF = k_Cos_OneEighthPi<float>;
+inline constexpr double k_Cos_OneEighthPiD = k_Cos_OneEighthPi<double>;
+
+inline constexpr float k_Cos_ThreeEighthPiF = k_Cos_ThreeEighthPi<float>;
+inline constexpr double k_Cos_ThreeEighthPiD = k_Cos_ThreeEighthPi<double>;
+
+inline constexpr float k_Sin_ThreeEighthPiF = k_Sin_ThreeEighthPi<float>;
+inline constexpr double k_Sin_ThreeEighthPiD = k_Sin_ThreeEighthPi<double>;
+} // namespace complex::Constants
```

Comment:
@imikejackson Is there a reason we didn't put these all in Numbers.hpp?  Or at least the numbers namespace if not the file itself? I feel like if I were looking for these I would not think to check for them here if there was a file called Numbers.hpp already.

Reply:
The Numbers.hpp is a replacement for a C++20 header so our thinking is that when we move to C++20 and we get Numbers.hpp for free we don't have to then move a bunch of constants to a new file. Doing this now alleviates that need later on.